github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -567,7 +567,7 @@ presubmits:
         name: ""
         resources:
           requests:
-            memory: 4Gi
+            memory: 8Gi
```

Comment:
nit: this is running unnested, so it shouldn't be required to double memory for it.